src/pages/guides/getting-started/nodejs/api-with-nextjs.mdx

```diff
@@ -1,71 +1,38 @@
 export const description =
   'Build a serverless backend for your Next.js app using Nitric framework for AWS, Google Cloud or Azure'
 
-# Serverless apps with Next.js and Nitric
-
-## What we'll be doing
-
-1. Create a To-do app with [Next.js](https://nextjs.org) backed by a Nitric API
-2. Define our Nitric resources
-3. Create handlers for the following API operations
-
-| **Method** | **Route**    | **Description**                                     |
-| ---------- | ------------ | --------------------------------------------------- |
-| `GET`      | /:listid/:id | Get a specific task by its task list Id and task Id |
-| `GET`      | /:listid     | Get a specific task list by its Id                  |
-| `GET`      | /            | List all task lists                                 |
-| `POST`     | /:listid     | Add new task to task list                           |
-| `POST`     | /            | Create a new task list                              |
-| `PATCH`    | /:listid/:id | Update a task                                       |
-| `DELETE`   | /:listid/:id | Delete a task                                       |
-| `DELETE`   | /:listid     | Delete a task list                                  |
-
-3. Set up a proxy for the Next.js API
-4. Run locally for testing
-5. Deploy backend to a cloud of your choice
-6. Deploy frontend to Vercel or Netlify
+# Next.js and Nitric example application
+
+This example application demonstrates a To-Do List built using [Next.js](https://nextjs.org) for the frontend and Nitric for the API.
 
 ## Prerequisites
 
-- [Node.js](https://nodejs.org/en/download/)
-- The [Nitric CLI](/guides/getting-started/installation)
-- An [AWS](https://aws.amazon.com), [GCP](https://cloud.google.com) or [Azure](https://azure.microsoft.com) account (_your choice_)
+Before getting started you'll want [Node.js](https://nodejs.org/en/download/) and the [Nitric CLI](/guides/getting-started/installation) installed. If you'd like to deploy the application to the cloud you'll also need an [AWS](https://aws.amazon.com), [Google Cloud](https://cloud.google.com) or [Azure](https://azure.microsoft.com) account. The code for the application will be the same regardless of the cloud you choose.
```

Comment:
Looks good. Do you think it should be suggested that a vercel/netlify account is created as well? This would also have the caveat that it's not necessary unless the frontend will be deployed.